src/components/product-tiles/ProductTiles.test.tsx

```diff
@@ -29,7 +29,7 @@ test('renders a list of tiles with correct properties', async () => {
   const secondTileName = await screen.findByText(
     `${mockProductsResponse.products[1]._source.name}`
   );
-  const tilePrices = await screen.findAllByText(/£/);
+  const tilePrices = await screen.findAllByText(/[£$€]/);
```

Comment:
This bit can be tricky if we support other locales. It's best to use the same approach you used to get the formatted price for a specific locale

Reply:
Ah yes that makes sense, I think I've changed the test to do that instead now

Reply:
I've committed the changes to this test file